<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" total="5" passed="3">
  <reporter-output>
  </reporter-output>
  <suite name="eBankingV1" duration-ms="66182" started-at="2021-09-14T07:24:19Z" finished-at="2021-09-14T07:25:25Z">
    <groups>
    </groups>
    <test name="BankingApp Test" duration-ms="66182" started-at="2021-09-14T07:24:19Z" finished-at="2021-09-14T07:25:25Z">
      <class name="com.enetBanking.testCases.TC_loginDDT_002">
        <test-method status="PASS" signature="setUp(java.lang.String)[pri:0, instance:com.enetBanking.testCases.TC_loginDDT_002@53b32d7]" name="setUp" is-config="true" duration-ms="4500" started-at="2021-09-14T12:54:19Z" finished-at="2021-09-14T12:54:24Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="loginTest(java.lang.String, java.lang.String)[pri:0, instance:com.enetBanking.testCases.TC_loginDDT_002@53b32d7]" name="loginTest" duration-ms="16201" started-at="2021-09-14T12:54:25Z" data-provider="LoginData" finished-at="2021-09-14T12:54:41Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[mngr353180]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[nerynYt]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method status="FAIL" signature="loginTest(java.lang.String, java.lang.String)[pri:0, instance:com.enetBanking.testCases.TC_loginDDT_002@53b32d7]" name="loginTest" duration-ms="5143" started-at="2021-09-14T12:54:42Z" data-provider="LoginData" finished-at="2021-09-14T12:54:47Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[mngr140789]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[AeUdYdf]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at com.enetBanking.testCases.TC_loginDDT_002.loginTest(TC_loginDDT_002.java:37)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method status="PASS" signature="loginTest(java.lang.String, java.lang.String)[pri:0, instance:com.enetBanking.testCases.TC_loginDDT_002@53b32d7]" name="loginTest" duration-ms="16056" started-at="2021-09-14T12:54:47Z" data-provider="LoginData" finished-at="2021-09-14T12:55:03Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[mngr353180]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[nerynYt]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method status="FAIL" signature="loginTest(java.lang.String, java.lang.String)[pri:0, instance:com.enetBanking.testCases.TC_loginDDT_002@53b32d7]" name="loginTest" duration-ms="5088" started-at="2021-09-14T12:55:03Z" data-provider="LoginData" finished-at="2021-09-14T12:55:08Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[mngr314567]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[AeUaYdE]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertTrue(Assert.java:42)
	at org.testng.Assert.assertTrue(Assert.java:52)
	at com.enetBanking.testCases.TC_loginDDT_002.loginTest(TC_loginDDT_002.java:37)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method status="PASS" signature="loginTest(java.lang.String, java.lang.String)[pri:0, instance:com.enetBanking.testCases.TC_loginDDT_002@53b32d7]" name="loginTest" duration-ms="16374" started-at="2021-09-14T12:55:08Z" data-provider="LoginData" finished-at="2021-09-14T12:55:25Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[mngr353180]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[nerynYt]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.enetBanking.testCases.TC_loginDDT_002@53b32d7]" name="tearDown" is-config="true" duration-ms="694" started-at="2021-09-14T12:55:25Z" finished-at="2021-09-14T12:55:25Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.enetBanking.testCases.TC_loginDDT_002 -->
    </test> <!-- BankingApp Test -->
  </suite> <!-- eBankingV1 -->
</testng-results>
